diff --git a/node_modules/react-native-leaflet-view/android/build.gradle b/node_modules/react-native-leaflet-view/android/build.gradle
index 6a41eb0..bf402d6 100644
--- a/node_modules/react-native-leaflet-view/android/build.gradle
+++ b/node_modules/react-native-leaflet-view/android/build.gradle
@@ -1,17 +1,16 @@
 buildscript {
-  // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['Leaflet_kotlinVersion']
+    def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['Leaflet_kotlinVersion']
 
-  repositories {
-    google()
-    jcenter()
-  }
+    repositories {
+        google()
+        mavenCentral() // Remplace jcenter()
+    }
 
-  dependencies {
-    classpath 'com.android.tools.build:gradle:3.2.1'
-    // noinspection DifferentKotlinGradleVersion
-    classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-  }
+    dependencies {
+        // On remonte un peu la version pour plus de stabilité avec ton Gradle récent
+        classpath 'com.android.tools.build:gradle:7.4.2'
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
 }
 
 apply plugin: 'com.android.library'
@@ -52,7 +51,6 @@ android {
 
 repositories {
   mavenCentral()
-  jcenter()
   google()
 
   def found = false
